### ISSUE TITLE
apVisit() arguments incorrect?

### ISSUE THREAD

**Issue by Spiffical** [jobovy/apogee#42]
In your README, when you discuss using apVisit, you mention how to pull a spectrum:
```
import apogee.tools.read as apread
# the three arguments are location ID, MJD, and fiber ID
spec = apread.apVisit(7439, 56763, 207, ext=1, header=False)
spec = apread.apVisit(7439, 56763, 207, ext=1, header=False)
```

I tried automating this with the allVisit catalog:
```
allVisit = apread.allVisit(rmcommissioning=True,main=True,ak=True, akvers='targ')
spec = apread.apVisit(allVisit['LOCATION_ID'][0], allVisit['MJD'][0], allVisit['FIBERID'][0], ext=1, header=False)
```
and received the error `IOError: File apVisit-r5-4264-55874-268.fits does not appear to exist on the server (as http://data.sdss3.org/sas/dr12/apogee/spectro/redux/r5/apo25m/4264/55874/apVisit-r5-4264-55874-268.fits)`

I checked this URL and none of the allVisit location IDs were listed there. It took me a few minutes to realize that it actually needed the plate ID instead of the location ID, i.e. `allVisit['PLATE'][0]` instead of `allVisit['LOCATION_ID'][0]`. Still got an error after changing this because your code queries the apo1m if the argument is a string, so I changed it to an int (after deleting whitespace). The final line that worked is:

`spec = apread.apVisit(int(allVisit['PLATE'][i].strip()), allVisit['MJD'][i], allVisit['FIBERID'][i], ext=1, header=False)`

Just thought you should be aware that it is the PLATE ID, not the LOCATION ID, that apVisit needs!

**Comment by jobovy** [comment]
Thanks for pointing this out! 

I'm cc-ing @mrawls, because she added the apVisit functionality to the code. I think we'll need to replace loc_id with plate_id and then I don't think there's a simple way to figure out which telescope is being used, so I think we need to add a keyword ``telescope=`` to specifiy the telescope for the visits (probably a good idea in general, to prepare for APOGEE-2S).

**Comment by mrawls** [comment]
Good catch! I don't have time to fix this right now, but I will try to get to it in the next few weeks. In the meantime I hope your creative workaround can help anyone else who runs into this problem.
